experimental/puppeteer-firefox/lib/Page.js

```diff
@@ -58,7 +58,7 @@ class Page extends EventEmitter {
     this._networkManager.setFrameManager(this._frameManager);
     this._eventListeners = [
       helper.addEventListener(this._session, 'Page.uncaughtError', this._onUncaughtError.bind(this)),
-      helper.addEventListener(this._session, 'Page.console', this._onConsole.bind(this)),
+      helper.addEventListener(this._session, 'Runtime.console', this._onConsole.bind(this)),
```

Comment:
consoleAPICalled?

Reply:
This dispatches not only console API calls, but also all warnings / violations that console emits.